src/components/organisms/OnBoarding/components/Buttons.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { BackgroundButton } from 'components/atoms';
+
+type Color = string | undefined;
+const BackgroundColor = (color: Color) => {
+  switch (color) {
```

Comment:
Switch for only 2 cases?

Reply:
Intuitively a UpperCamelCase variable is a Class/Interface and here it's a function.
Function name could be smthing like `getBackgroundColor`.
A `Color` that can be the string `inactive` is not very intuitive.

Reply:
My bad, the idea is to clean all the Buttons and have some props for everything (color, size, etc.)

What's the best way until then?

Reply:
Until then just list specific strings like `inactive` (like an enum) in your type :
```
type Color = 'inactive' | string | undefined;
```
Is color the best name for these ?

You could prepare the ground and do smthing like this:
```
type ButtonState = 'inactive' | undefined;
type Color = string | ButtonState;
```

Reply:
I think you lost me. 

How to use that?

Reply:
`Ctrl+V` then `Ctrl+V` to replace `type Color = string | undefined;`

Reply:
and you mean that I keep the switch?

I don't get the point but m'kay ^^

Reply:
I feel like I'm stealing a kid toy... :car: 
"You may keep the `switch` this time. But next time you have only 2 cases, you must promesse me you'll use an `if ... else`."

Reply:
@lutangar I changed that yesterday to put a if statement, you didn't see my changes ?

Reply:
> here @lutangar @newick
> reassure me, you also see the translations?

I see the translations, yep.
Oh and yeah there's a `if`, didn't checked it, my bad 🤦